modules/olmv1-updating-an-operator.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_content-type: PROCEDURE
+
+[id="olmv1-updating-an-operator_{context}"]
+= Updating an Operator
+
+You can update your Operator by manually editing your Operator's custom resource (CR) and applying the changes.
+
+.Prerequisites
+
+* You have a catalog installed.
+* You have an Operator installed.
+
+.Procedure
+
+. Inspect your Operator's package contents to find which channels and versions are available for updating by running the following command:
++
+[source,terminal]
+----
+$ oc get package <catalog_name>-<operator_name> -o yaml
+----
++
+.Example command
+[source,terminal]
+----
+$ oc get package redhat-operators-quay-operator -o yaml
+----
+
+. Edit your Operator's CR to update the version to `3.9.1`, as shown in the following example:
```

Comment:
@michaelryanpeter I have new comments for your updated.
you remove channel information, but this example is following the installation section. for installation section, the example has channel, and but when I update it, the example has no channel. it is a little strange.
I think we should keep the same rule.

and the example output still keep channel information. (refer to https://github.com/openshift/openshift-docs/pull/66036/files#diff-3a4664f5b5ccef6dd8762bcc6bc754af9267fa277045a6f9aeabb7807fb32087R102)

so,
1, if you want to remove channel information, please update the output and update the installation section as well.
or
2, if you move channel information back here, no need to change others.

Reply:
if you still want to remove channel, here is the output.
```console
[root@preserve-olm-env2 olmv1review]# oc get operator.operators.operatorframework.io/quay-example -o yaml
apiVersion: operators.operatorframework.io/v1alpha1
kind: Operator
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"operators.operatorframework.io/v1alpha1","kind":"Operator","metadata":{"annotations":{},"name":"quay-example"},"spec":{"packageName":"quay-operator","version":"3.9.1"}}
  creationTimestamp: "2023-10-19T02:39:05Z"
  generation: 2
  name: quay-example
  resourceVersion: "138637"
  uid: 480be93d-e325-4831-ac4a-8bfd451ef3bd
spec:
  packageName: quay-operator
  version: 3.9.1
status:
  conditions:
  - lastTransitionTime: "2023-10-19T02:39:05Z"
    message: resolved to "registry.redhat.io/quay/quay-operator-bundle@sha256:4864bc0d5c18a84a5f19e5e664b58d3133a2ac2a309c6b5659ab553f33214b09"
    observedGeneration: 2
    reason: Success
    status: "True"
    type: Resolved
  - lastTransitionTime: "2023-10-19T02:39:27Z"
    message: installed from "registry.redhat.io/quay/quay-operator-bundle@sha256:4864bc0d5c18a84a5f19e5e664b58d3133a2ac2a309c6b5659ab553f33214b09"
    observedGeneration: 2
    reason: Success
    status: "True"
    type: Installed
  installedBundleResource: registry.redhat.io/quay/quay-operator-bundle@sha256:4864bc0d5c18a84a5f19e5e664b58d3133a2ac2a309c6b5659ab553f33214b09
  resolvedBundleResource: registry.redhat.io/quay/quay-operator-bundle@sha256:4864bc0d5c18a84a5f19e5e664b58d3133a2ac2a309c6b5659ab553f33214b09
```

Reply:
Oops. I thought I cleaned up the output. Thank you. I will fix it.

I removed the channel information because engineering mentioned that there is no point in specifying both a channel and a version at the same time.